learn/views.py

```diff
@@ -353,3 +356,73 @@ def autocompleteSuggestionTopic(request):
 			'list': filter_topic,
 		}
 		return JsonResponse(data)
+
+
+url_entires = {}
+
+def youtubeResource(request):
```

Comment:
Add relevant comments which will make it easy for other to understand logic behind code.